### ISSUE TITLE
[feature request] convert stereogram to stereo picture

### ISSUE THREAD

**Issue by tufeixp** [gnudles/stereograma#2]
Why not add a feature to convert stereogram to anaglyph or sbs stereographic picture and view it on a 3d TV?

**Comment by gnudles** [comment]
You mean creating sbs stereographic instead of autostereogram, not *from* autostereogram

**Comment by tufeixp** [comment]
> You mean creating sbs stereographic instead of autostereogram, not _from_ autostereogram

Not exactly, there should be a way for a 3d renderer to load any stereogram and render it into two view stereo, eg. for 3d shutter/polarized glasses and head mounted glasses.

**Comment by gnudles** [comment]
The only way to do this is to use 3D Reconstruction algorithm, which I am not sure that exists. And even if there was any, it is out of scope of this project. I am currently considering adding video input support.
Exporting depth maps into stereo-pair is something that I might consider
